apps/condo/migrations/20230912113650-0313_auto_20230912_0636.js

```diff
@@ -0,0 +1,50 @@
+// auto generated by kmigrator
+// KMIGRATOR:0313_auto_20230912_0636:IyBHZW5lcmF0ZWQgYnkgRGphbmdvIDMuMi41IG9uIDIwMjMtMDktMTIgMDY6MzYKCmZyb20gZGphbmdvLmRiIGltcG9ydCBtaWdyYXRpb25zLCBtb2RlbHMKCgpjbGFzcyBNaWdyYXRpb24obWlncmF0aW9ucy5NaWdyYXRpb24pOgoKICAgIGRlcGVuZGVuY2llcyA9IFsKICAgICAgICAoJ19kamFuZ29fc2NoZW1hJywgJzAzMTJfdXNlcnJpZ2h0c3NldGhpc3RvcnlyZWNvcmRfYW5kX21vcmUnKSwKICAgIF0KCiAgICBvcGVyYXRpb25zID0gWwogICAgICAgIG1pZ3JhdGlvbnMuUmVtb3ZlRmllbGQoCiAgICAgICAgICAgIG1vZGVsX25hbWU9J2IyYmFwcCcsCiAgICAgICAgICAgIG5hbWU9J2lzR2xvYmFsJywKICAgICAgICApLAogICAgICAgIG1pZ3JhdGlvbnMuUmVtb3ZlRmllbGQoCiAgICAgICAgICAgIG1vZGVsX25hbWU9J2IyYmFwcGhpc3RvcnlyZWNvcmQnLAogICAgICAgICAgICBuYW1lPSdpc0dsb2JhbCcsCiAgICAgICAgKSwKICAgICAgICBtaWdyYXRpb25zLkFkZEZpZWxkKAogICAgICAgICAgICBtb2RlbF9uYW1lPSdiMmJhcHAnLAogICAgICAgICAgICBuYW1lPSdnbG9iYWxVcmwnLAogICAgICAgICAgICBmaWVsZD1tb2RlbHMuVGV4dEZpZWxkKGJsYW5rPVRydWUsIG51bGw9VHJ1ZSksCiAgICAgICAgKSwKICAgICAgICBtaWdyYXRpb25zLkFkZEZpZWxkKAogICAgICAgICAgICBtb2RlbF9uYW1lPSdiMmJhcHBoaXN0b3J5cmVjb3JkJywKICAgICAgICAgICAgbmFtZT0nZ2xvYmFsVXJsJywKICAgICAgICAgICAgZmllbGQ9bW9kZWxzLlRleHRGaWVsZChibGFuaz1UcnVlLCBudWxsPVRydWUpLAogICAgICAgICksCiAgICBdCg==
+
+exports.up = async (knex) => {
+    await knex.raw(`
+    BEGIN;
+--
+-- Remove field isGlobal from b2bapp
+--
+ALTER TABLE "B2BApp" DROP COLUMN "isGlobal" CASCADE;
+--
+-- Remove field isGlobal from b2bapphistoryrecord
```

Comment:
By what logic is the fact that this mini-application is now determined to be global?

Reply:
If it done with `globalUrl` prop, I suppose you should take into account already existing records in the database

Reply:
If it has `globalUrl`. Miniapp can have `appUrl` to connect and configure miniapp and `globalUrl` for background work in service

Reply:
> If it done with `globalUrl` prop, I suppose you should take into account already existing records in the database

updated in migration

Reply:
> > If it done with `globalUrl` prop, I suppose you should take into account already existing records in the database
> 
> updated in migration

That's what I'm talking about. Looks good now